pkg/geo/geogen/geogen.go

```diff
@@ -208,7 +208,12 @@ func RandomMultiPoint(
 	// 10% chance to generate an empty multipoint (when num is 0).
 	num := rng.Intn(10)
 	for i := 0; i < num; i++ {
-		if err := ret.Push(RandomPoint(rng, randomBounds, srid, layout)); err != nil {
+		// TODO(ayang): permit EMPTY collection item once GEOS is patched
```

Comment:
can you replace ayang with an issue number?
unfortunately you're not here forever :3